react/components/SearchMetadata/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+import WithStaticRouter from 'react/hocs/WithStaticRouter';
+import Grid from 'react/components/UI/Grid';
+import HeaderMetadataContainer from 'react/components/UI/HeaderMetadata/HeaderMetadataContainer';
+
+import SearchMetadataBreadcrumb from './components/SearchMetadataBreadcrumb';
+import SearchMetadataModeView from './components/SearchMetadataModeView';
```

Comment:
I'm cool with adding aliases for imports. (I like to copy paths of files out of my editor with shortcuts). Aliases or absolute paths are preferred though so that we can just copy imports indiscriminately between files.